src/Demo/suggestionsStore.ts

```diff
@@ -0,0 +1,41 @@
+import { SuggestionMeta } from "../ApiModules/autocompleteSession";
+
+const INITIAL_SUGGESTIONS: SuggestionMeta[] = [
+  { suggestion: 'Nice to meet you!', type: 'default', baseClass: 'suggestion', expandedSuggestion: '', userAdded: false },
```

Comment:
What other suggestions should we have in demo?

Reply:
I feel like this is fine for now - we should update the tutorial with a variable suggestion, but in a later story since it's really only on gmail and user's don't have much control over the vals rn